.github/workflows/release-prod.yml

```diff
@@ -0,0 +1,35 @@
+name: Production release PR
+run-name: Production release PR
+
+on:
+  push:
+    branches:
+      - main
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  release:
+    name: Release
+    #    if: ${{ github.repository == 'clerkinc/javascript' }}
```

Comment:
❓ Is this a leftover?

Reply:
We need this in order to prevent actions to run against forked repositories, however I've commented it out for now so I can test on my own forked repo.

Will enable this check again once this is merged :)